cityofaustin/atd-airflow
# Remove CR3 and non-CR3 crash location housekeeping ETLs

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/13190

This removes the DAG that ran the crash re-association scripts that we have turned off now that DB triggers are handling crash location associations.

## Associated repo
https://github.com/cityofaustin/atd-vz-data/pull/1304

## Testing

**Steps to test:**
Spin up the local stack and see that the `vz_reassociate_missing_locations` DAG is no longer listed

---
#### Ship list
